cmd/pkiaas/main.go

```diff
@@ -14,10 +23,53 @@ func getPort() string {
 	return ":8080"
```

Comment:
Do we want to default to port 8080 if we're automatically enabling self-signed TLS instead of going without? I would say if not 443, we should change this to 4443 or something... 8443?  I don't know.  What do you think, @AndrewCopeland & @rcobbins?

Reply:
I think we should default to 443. @rcobbins

Reply:
Fine by me, I'll change it in GitHub